src/training/flava_eval/finetune_glue.py

```diff
@@ -223,9 +269,8 @@ def train_one_epoch(model, data, epoch, optimizer, scheduler, early_stop, device
         label = batch["label"].to(device)
 
         logits = model(text)
-        logits = logits.view(-1)
-        label = label.view(-1).float()
-        loss = nn.functional.binary_cross_entropy_with_logits(logits, label)
+        loss_fn = nn.functional.mse_loss if args.task_name == "stsb" else nn.functional.cross_entropy
```

Comment:
this is repeated code, make a method called `get_loss_fn(task_name)` or something like that and call it to get the loss function